### ISSUE TITLE
Vort Crash Bug

### ISSUE THREAD

**Issue by Thebonedude** [urnotnick/impulse-issues#66]
**Describe the bug**
Simply just vorts crash in some circumstences, will experiment with it more

**To reproduce**
Steps to reproduce the behaviour:
1. Works only on vortessenced vorts (as far as I knew)
2. Two of us were running in sewers
3. Crashed

**Expected behaviour**
I excepcted to not crash

1.10.3

**Lua error**
Executable: C:\Program Files (x86)\Steam\steamapps\common\GarrysMod\bin\win64\gmod.exe

-Lua Stack Traces-
==================
  Client
    Not in Lua call OR Lua has panicked

  Server
    Lua Interface = NULL

  MenuSystem
    Not in Lua call OR Lua has panicked

**Additional context**
It cannot be a client-side error considering we were two (maybe three) to have crashed at the exact same place in sewers

**Comment by urnotnick** [comment]
do you have a .mdmp file of your crash?

**Comment by Thebonedude** [comment]
Yes, I do, I'll send a link through discord because apparently github doesn't support those files
https://discord.com/channels/917452949117091850/917452949117091853/972617799845433344
